queue/driver/database.go

```diff
@@ -0,0 +1,54 @@
+package driver
+
+import (
+	"github.com/goravel/framework/contracts/database/orm"
+	"github.com/goravel/framework/contracts/queue"
+)
+
+type Database struct {
+	connection string
+	db         orm.Orm
+}
+
+func NewDatabase(connection string, db orm.Orm) *Database {
+	return &Database{
+		connection: connection,
+		db:         db,
+	}
+}
+
+func (d *Database) ConnectionName() string {
+	return d.connection
+}
+
+func (d *Database) Push(job queue.Job) error {
+	return nil
+}
+
+func (d *Database) Bulk(jobs []queue.Job) error {
+	return nil
+}
+
+func (d *Database) Later(job queue.Job, delay int) error {
+	return nil
+}
+
+func (d *Database) Pop() (queue.Job, error) {
+	return nil, nil
+}
```

Comment:
Need to study how to run the pop-up tasks. Consider adding a `go func` scheduled listener?
Laravel use `queue:work` artisan command.